src/applications/mhv-medical-records/tests/util/pdfHelpers/blueButton.unit.spec.jsx

```diff
@@ -14,7 +14,7 @@ describe('generateBlueButtonData', () => {
   it('should return an empty array when no records are provided', () => {
     const result = generateBlueButtonData({}, []);
     expect(result).to.be.an('array');
-    expect(result.length).to.equal(11);
+    expect(result.length).to.equal(9);
```

Comment:
Why did this value change from 11 to 9 if you didn't modify any text data or change any logic?

Reply:
Got it -- the number of sections was reduced by this change (blank sections don't appear).